addons/gut/plugin_control.gd

```diff
@@ -27,83 +27,87 @@
 #
 # ##############################################################################
 # This is the control that is added via the editor.  It exposes GUT settings
-# through the editor and delays the creation of the GUT instance until
+# through the editor and delays the creation of the GUT instantiate until
 # Engine.get_main_loop() works as expected.
 # ##############################################################################
-tool
+@tool
 extends Control
 
 # ------------------------------------------------------------------------------
 # GUT Settings
 # ------------------------------------------------------------------------------
-export(String, 'AnonymousPro', 'CourierPrime', 'LobsterTwo', 'Default') var _font_name = 'AnonymousPro'
-export(int) var _font_size = 20
-export(Color) var _font_color = Color(.8, .8, .8, 1)
-export(Color) var _background_color = Color(.15, .15, .15, 1)
+@export var _font_name = 'AnonymousPro' # (String, 'AnonymousPro', 'CourierPrime', 'LobsterTwo', 'Default')
```

Comment:
This could use a `# !4.0 comment`.  It appears that the value hints are either done differently or doo not exist.  Adding a comment here and noting that there are others in here that were altered will be helpful.